openrtb_ext/imp_ix.go

```diff
@@ -4,4 +4,5 @@ package openrtb_ext
 type ExtImpIx struct {
 	SiteId string `json:"siteId"`
 	Size   []int  `json:"size"`
+	Sid    string `json:"sid"`
```

Comment:
Please [params test](https://github.com/prebid/prebid-server/blob/master/adapters/ix/params_test.go) for this new parameter.